LULESH/lulesh.cc

```diff
@@ -2625,12 +2626,13 @@ void DumpMultiblockObjects(DBfile *db, char basename[], int numRanks)
   int *varTypes;
   int ok = 0;
   // Make sure this list matches what's written out above
-#ifdef USE_ADAPTIVE_SAMPLING
+  // All variables related to adaptive sampling MUST come AFTER the others
```

Comment:
This is a bit cheesy, but I'll try and clean it up as we start running benchmarks.